consensus/polybft/bridge.go

```diff
@@ -63,6 +66,54 @@ func newBridge(runtime Runtime,
 	return bridge, nil
 }
 
+// initBridgeEventRelayer initializes bridge event relayer
+// if not enabled, then a dummy bridge event relayer will be used
+func (b *bridge) initBridgeEventRelayer(
+	eventProvider *EventProvider,
+	runtimeConfig *runtimeConfig,
+	logger hclog.Logger) error {
+	if runtimeConfig.consensusConfig.IsRelayer {
+		txRelayerMap := make(map[uint64]txrelayer.TxRelayer)
```

Comment:
I would move all this logic for initializing a `relayer` to the `relayer` struct. Basically, I would just call `newRelayer` constructor here, pass everything needed to create it and init its trackers, and that is all. The constructor function would then set up everything and call `initTrackers`.

Also make sure to have a `Close` function on the relayer, so we can stop its trackers. And call it in the `bridge.Close()` function.